common/model/transfer.c

```diff
@@ -167,6 +167,8 @@ transfer_t* transfer_data_new(flex_trit_t const* const address,
     transfer->address = address;
     transfer->value = 0;
     transfer->tag = tag;
+    flex_trits_slice(transfer->obsolete_tag, NUM_TRITS_TAG, tag, NUM_TRITS_TAG,
```

Comment:
`tag` and `obsolete_tag` have the same  size, please use `memcpy`. `flex_trits_slice` is really inefficient.